cmd/scollector/collectors/processes_linux.go

```diff
@@ -1,18 +1,29 @@
 package collectors
 
 import (
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
```

Comment:
Shouldn't be moved to top, these are always after a space after the normal packages. Using goimports?